src/generator/writers/writeExpression.ts

```diff
@@ -471,7 +471,7 @@ export function writeExpression(f: ASTExpression, ctx: WriterContext): string {
     if (f.kind === 'op_static_call') {
 
         // Check global functions
-        if (GlobalFunctions[f.name]) {
+        if (Object.prototype.hasOwnProperty.call(GlobalFunctions, f.name)) {
```

Comment:
Writing `Object.prototype.hasOwnProperty.call` whenever we need to access a key of a simple map does not look to enticing. I'd propose to declare `GlobalFunctions` a `Map` (as well as the other globals):

```ts
export const GlobalFunctions:Map<string, AbiFunction> = new Map([
    ["ton", { ... }],
    ["pow", { ... }],
    ...
]);
```

 and then use the `.get(...)` method for access here and below. In this case `GlobalFunctions.get("toString")` is guaranteed to return `undefined`.

Reply:
In this case, it should be `.has` instead of `.get`, of course

Reply:
This suggestions looks good. There are actually many `Map` objects being used in other parts of the code, so it will fit well.